build_tools/buildkite/github/post_benchmarks_as_pr_comment.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+# Lint as: python3
```

Comment:
Having python3 in the first line is sufficient, so no need for the extra "Lint as"

```suggestion
```